pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -80,6 +85,7 @@ type NodeController struct {
 	recorder           record.EventRecorder
 	registerRetryCount int
 	podEvictionTimeout time.Duration
+	nodeStatusMap      map[string]NodeStatusData
```

Comment:
Please add a comment what this is storing. Probably the most accurate thing to say is "local time on NodeController corresponding to most recent LastProbeTime"